### ISSUE TITLE
save layer error

### ISSUE THREAD

**Issue by AsdrtChen** [google/earthenterprise#881]
Dears , 

I build a Map Layer . 
And I change the icon .
but I can't save if I change the icon 
![image](https://user-images.githubusercontent.com/30334676/41396798-9e351024-6fe4-11e8-8f87-4d39704f3d57.png)

And I already chmod + chown those file .
it has the same error message 
how do I fix this problem 

thanks

**Comment by ericwhiteau** [comment]
I have also just run into this issue,
Note that the 3D  earth build was successful with the same Icon file names.

Happens in 2D with both standard and custom icons.  Definitly not a permissions issue,  I have set read all  and as stated it works in 3d build with the same icons.

**Comment by tst-ccamp** [comment]
I missed this bug before. We'll start taking a look

**Comment by ericwhiteau** [comment]
With this issue in 2D maps,  I need to understand my options.
I was migrating an Earth project to a new infrastructure.   Currently runs 5.1.3  the last google release.
I did testing with OpenGee and built full 3D solution (after full test suite against 3d globe).
It was only then that I addressed the 2d maps side.  which is failing due to able issue.

Is there a way to roll back Open GEE to 5.1.3?
I get the message. 
/gevol/assets  is configured for "5.2.1"  you have 5.1.3  need to upgrade.....

Is there a valid way to downgrade to 5.1.3?    changing the version in .config file doesnt work as I tested that.
Attempting to avoid build from scratch as there is 30days of build time.

not sure what sort of timeline is on bug fixes against 5.2.4?
thanks in advance for any comments.

**Comment by ericwhiteau** [comment]
PS   I note that   the library   "SkImageDecoder::DecodeFile"

is used in 2d maps and not in 3d client.

this seems to have a valid path passed to it but returns error.

**Comment by tst-sbayard** [comment]
@tst-mbiancalana and I started looking at this last week, and we believe we've identified the issue. @ericwhiteau, good detective work tracking it down to SkImageDecoder::DecodeFile". The underlying problem is that, because of some fancy linking trickery in combination with how SCons executes the SConscript files, the codecs for the icons don't get linked in, so they don't even exist in the Fusion install ☹️ 
Hence, Fusion doesn't know how to decode them for 2d map layers.

 We're currently working to determine the best fix for this.

@AsdrtChen and @ericwhiteau thanks for reporting this and doing some digging on it.
